runtime/onert/backend/gpu_cl/open_cl/Operations.h

```diff
@@ -370,6 +371,7 @@ struct ReLUAttributes
   float alpha = 0;
 };
 
+#if 0
```

Comment:
Even if blocked with if 0 because of TC not ready, side effects, or NYI, how about put a small comment ? e.g) #if 0 //NYI

Reply:
I modify it.